### PR TITLE
Shopping Bag

### PR DESCRIPTION
**Pull request by rockondiana0416**
I left the CSS for the shopping bag empty. I wasn't sure if I should do an import or includes. I would think an import. But the checkout is separate from the rest of the site. So I'm trying to wrap my head around styling it. I've looked at other websites and there is very little CSS. Then the checkout window is usually a pop up to the right side of the screen. Let me know what you think.

### CODE REVIEW

**Comment by thompson-ele** [comment]
@rockondiana0416 So I think what you're thinking of is called a side rail. You'd have the products added to a small bag that popups out from the right hand side. To reduce complexity, I'd prefer if we have a dedicated shopping bag page. Something like Nordstrom's shopping bag where we have a list of products being purchased, followed by a total and button to continue the checkout process: https://secure.nordstrom.com/shopping-bag

This will allow us to do everything in PHP rather than having to use JavaScript which is not the focus of this class. I'm including a screenshot of an example I found on the interwebs that may be good for what we're doing:
Screen Shot 2019-11-26 at 8 20 04 PM

Let me know if you have any other questions!

File: model/shopping-bag.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
+<?php ?>
```
Comment (thompson-ele) [inline model/shopping-bag.php]: Please move this file out of the model folder :)

**Review by thompson-ele — CHANGES_REQUESTED** [review]
Make sure the shopping-bag.php file is outside of the model folder. For the shopping-bag.scss file I'd recommend moving it into the styleguide/modules folder and you would add an import in modules.scss so that the CSS you write in shopping-bag.scss gets compiled.

**Comment by rockondiana0416** [comment]
Hi Ele! Leo and I made changes to the shopping-bag. Patrick is taking a look at the css. Let me know what you think.

**Review by thompson-ele — APPROVED** [review]
Nice :)

**Pull request merged**